vars/sectionBuildAndTest.groovy

```diff
@@ -101,6 +101,7 @@ def call(params) {
 
             pcr.callAround('dockerbuild') {
               timeoutWithMsg(time: 30, unit: 'MINUTES', action: 'Docker build') {
+                writeFile file: '.dockerignore', text: libraryResource('uk/gov/hmcts/.dockerignore_build')
```

Comment:
what if the docker ignore file already exists?

Reply:
Where? I looked for one.

Reply:
we never got a reliable fix, we turned off the files that were causing the issue in some builds which helped but sometimes people still hit it.

if this doesn't work I would either stop removing the file after use, or only remove it after the parallel stages have finished